operations/operations/mlt-cadence/README.md

```diff
@@ -252,60 +134,60 @@ The following should happen within a 2 week period, with 1-2 iterations in each
 **T-Minus 2 Weeks to announcement.**
 
 * At Monthly Strategic Meeting:
-  * WIP VPMOMs, quarterly plan and proposed orgs reviewed by MLT
-  * MLT shares feedback on each VPMOM and quarterly plan
+  * WIP V2MOMs, quarterly plan and proposed orgs reviewed by MLT
+  * MLT shares feedback on each V2MOM and quarterly plan
   * Cross-department dependencies documented
   * Draft agenda
     * Review of MLT Plan Process
-    * Company VPMOM & Q2 plan
-    * Sales VPMOM, Q2 plan and proposed org
-    * R&D VPMOM, Q2 plan and proposed org
-    * Marketing VPMOM, Q2 plan and proposed org
-    * Customer Success VPMOM, Q2 plan and proposed org
-    * Finance VPMOM, Q2 plan and proposed org
-    * People VPMOM, Q2 plan and proposed org
+    * Company V2MOM & Q2 plan
+    * Sales V2MOM, Q2 plan and proposed org
+    * R&D V2MOM, Q2 plan and proposed org
+    * Marketing V2MOM, Q2 plan and proposed org
+    * Customer Success V2MOM, Q2 plan and proposed org
+    * Finance V2MOM, Q2 plan and proposed org
+    * People V2MOM, Q2 plan and proposed org
 * MLT socializes updates to high-level plan with their departments
 
 **Finalize Plan**
 
 **T-Minus 1 Week to announcement**
 
-* Company and VPMOMs updated given feedback
+* Company and V2MOMs updated given feedback
 * Financial plan adjusted
 * Headcount approvals published
 
 **Plan Shared**
 
 **Week of announcement**
 
-* Company VPMOM shared at all-hands
-* Department VPMOMs are posted
+* Company V2MOM shared at all-hands
+* Department V2MOMs are posted
 * MLT presents company plan to their departments
 
 ## Quarterly Business Review \(QBR\)
 
-Each quarter we review process towards our VPMOMs and discuss achievements and opportunities.
-
-#### QBR VPMOM
-
-TBD
+Each quarter we review process towards our V2MOMs and discuss achievements and opportunities.
 
 #### Quarterly Business Review
```

Comment:
I think this may be extraneous as there is already a QBR heading (H2) above it.

Reply:
I agree. We can remove the duplication.